testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -567,6 +567,31 @@ impl BitcoinRegtestController {
         result_vec
     }
 
+    /// Checks if there is a default wallet with the name of "".
+    /// If the default wallet does not exist, this function creates a wallet with name "".
+    pub fn create_wallet_if_dne(&self) {
```

Comment:
Because these new methods are only called via tests, can you mark them all `cfg(test)`?